src/main/java/core/basesyntax/validation/UserValidatorImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.validation;
+
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+
+public class UserValidatorImpl implements UserValidator {
+    public void validate(User user) throws UserValidationException {
+        if (user == null) {
+            throw new UserValidationException("User can't be null");
+        }
+        if (user.getLogin() == null || user.getLogin().length() < 6) {
+            throw new UserValidationException("Login must be more than 6 characters");
+        }
+        if (user.getPassword() == null || user.getPassword().length() < 6) {
+            throw new UserValidationException("Password must be more than 6 characters");
```

Comment:
Similarly, the exception message for the password validation should be 'Password must be at least 6 characters'.